### PR TITLE
Implement organizations api

### LINKED ISSUE #24: Implement Organizations API resource
**Description**
Implement the API endpoints related to the organization resource and provide test coverage, as an example you might take a look at #6 

**Additional context**
The API reference for refunds can be found here: https://docs.mollie.com/reference/v2/organizations-api/get-organization

### PR DESCRIPTION
**Pull request by ducnt114**
## Description

Implement organizations api

## Motivation and context

closes #24 

## How has this been tested?

I added unit test

## Types of changes

What types of changes does your code introduce? Put an `x` in all the boxes that apply:
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

Go over all the following points, and put an `x` in all the boxes that apply.

Please, please, please, don't send your pull request until all of the boxes are ticked. Once your pull request is created, it will trigger a build on our continuous integration server to make sure your [tests and code style pass](https://help.github.com/articles/about-required-status-checks/).

- [x] I have read the **[CONTRIBUTING](CONTRIBUTING.md)** document.
- [x] My pull request addresses exactly one patch/feature.
- [x] I have created a branch for this patch/feature.
- [x] Each individual commit in the pull request is meaningful.
- [x] I have added tests to cover my changes.
- [x] If my change requires a change to the documentation, I have updated it accordingly.

If you're unsure about any of these, don't hesitate to ask. We're here to help!

### CODE REVIEW

File: mollie/organizations.go [hunk 1]
```diff
@@ -0,0 +1,74 @@
+package mollie
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+)
+
+// Organization describes a organization detail
```
Comment (VictorAvelar) [inline mollie/organizations.go]: Just some quick fix as go report card tracks grammar and misspelling
```suggestion
// Organization describes an organization detail
```

File: mollie/organizations.go [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+	Refunds       URL `json:"refunds,omitempty"`
+	Settlements   URL `json:"settlements,omitempty"`
+	Documentation URL `json:"documentation,omitempty"`
+}
+
+// OrganizationsService instance operates over organization resources
+type OrganizationsService service
+
+// Get retrieve an organization by its id.
+func (is *OrganizationsService) Get(id string) (i Organization, err error) {
```
Comment (VictorAvelar) [inline mollie/organizations.go]: You should return a pointer to the organization.

Also `is` is not descriptive for organization service I will rather use `os` or something else, IDK if os will be conflictive naming with the standard library `os` package.
```suggestion
func (os *OrganizationsService) Get(id string) (i *Organization, err error) {
```

File: mollie/organizations.go [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+		return
+	}
+	if err = json.Unmarshal(res.content, &i); err != nil {
+		return
+	}
+	return
+}
+
+// GetCurrent retrieve the currently authenticated organization
+func (is *OrganizationsService) GetCurrent() (i Organization, err error) {
```
Comment (VictorAvelar) [inline mollie/organizations.go]: Same as the previous comment

File: mollie/organizations_test.go [hunk 4]
```diff
@@ -0,0 +1,59 @@
+package mollie
```
Comment (VictorAvelar) [inline mollie/organizations_test.go]: You are only testing the happy path, can you please also add some tests for the errors returned?

If you don't know how to do it, probably is worth to take a look at the tests for `payments`

**Comment by ducnt114** [comment]
I added a commit to resolve your comment @VictorAvelar , please review.

**Pull request merged**